src/components/submission-common.js

```diff
@@ -95,15 +97,16 @@ export const recordsTable = props => {
     return records.length && _.every(k => _.includes(k, selectedIds), recordIds)
   }
 
-  const resizeColumn = (delta, columnName) => {
-    window.alert(`column resizing currently disabled (${delta}, ${columnName}`)
+  const resizeColumn = (currentWidth, delta, columnKey) => {
+    setDataTableColumnWidths(_.set(columnKey, currentWidth + delta))
   }
 
-  const columnWidth = 300
+  const withDataTableNamePrefix = columnName => `${selectedDataTable.name}/${columnName}`
 
   return h(AutoSizer, [({ width, height }) => {
     return h(GridTable, {
       'aria-label': `${selectedDataTable.name} data table`,
+      ref: dataTableRef,
```

Comment:
for my own education: what does this property do? is it easy enough to explain in a comment?

Reply:
So from what I understand, `useRef()` is a hook and it returns a mutable object. The initial value for the object is stored in `current` attribute. We can pass the reference to this object using `ref` variable and what this means is that whenever the corresponding DOM node's value changes, react will update the `.current` attribute for the ref. 
Reference: https://reactjs.org/docs/hooks-reference.html#useref